internal/provider/models/alerts/absence.go

```diff
@@ -0,0 +1,178 @@
+package alerts
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	. "github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	. "github.com/mezmo/terraform-provider-mezmo/internal/client"
+	. "github.com/mezmo/terraform-provider-mezmo/internal/provider/models/modelutils"
+)
+
+const ALERT_TYPE_ABSENCE = "absence"
+
+type AbsenceAlertModel struct {
```

Comment:
Painfully, these are all the same except for `absence` doesn't have `conditional`.  The use of a base model that's extended didn't work when TF instantiates it.  It feels really repetitive to repeat all this for a single field's difference, but that appears to be the "Go way" to do it.

Reply:
Yes, go would rather have you repeat yourself instead of difficult to trace inheritance / code reuse logic. I appreciate that position when debugging code but it can be frustrating when writing code.